action.yml

```diff
@@ -0,0 +1,139 @@
+# Copyright 2024 Google LLC
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
thanks for taking care of making sure this is there :) would not hurt to also add a LICENSE file to the repo IMO, as this is also something that GitHub will then automatically surface in their UI etc.

Reply:
cool, added